### PR TITLE
task-2 for 1st review

### PR DESCRIPTION
**Pull request by Maksym-Lizohub**
task-2 for 1st review

### CODE REVIEW

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                &:focus {
+                    color: @color-maksym__main-gray;
+                }
+            }
+        }
+    }
+}
+
+/* 
+.hero {
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Don't leave commented code, it has a bad impact on site performance

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            text-transform: capitalize;
+        }
+        &__item {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+        }
+        &__image {
+            display: flex;
+            flex-direction: column;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use flex mixins

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    .brands {
+        &__title {
+            position: absolute;
+            top: 5px;
+            z-index: 1;
+            background-color: white;
+            margin: 0;
+            padding: 0 5px 0 5px;
+            line-height: 30px;
+            font-size: 26px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use font-size mixin, check it here:
/lib/web/css/source/lib/_utilities.less : line 53

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+        position: relative;
+    }
+    .brands {
+        &__title {
+            position: absolute;
+            top: 5px;
+            z-index: 1;
+            background-color: white;
+            margin: 0;
+            padding: 0 5px 0 5px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use indent variables like @indent__xs;

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+        .lib-visually-hidden();
+    }
+}
+
+/* SECTION__BRANDS */
+.section__brands {
+    .row-full-width-inner {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use flex mixins

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+            width: 100%;
+            height: 100%;
+            margin: 0;
+            &:not(.pagebuilder-poster-overlay) {
+                width: 100%;
+            }
+            .pagebuilder-collage-content {
+                width: 100%;
+                display: flex;
+                justify-content: space-between;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use flex mixins

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+            }
+        }
+        [data-element="content"] {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use flex mixins

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 8]
```diff
@@ -0,0 +1,233 @@
+h1 {
+    .lib-visually-hidden();
+}
+.section.section__slider {
+    padding-bottom: 30px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use indent variables, check for:
```
@indent__xs;
@indent__s;
@indent__m;
@indent__l;
@indent__xl;
```

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+}
+
+.section {
+    padding-bottom: 30px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use indent variables

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 10]
```diff
@@ -0,0 +1,233 @@
...
+        [data-element="content"] {
+            padding-left: 17px;
+        }
+
+        .slick-dots {
+            position: absolute;
+            bottom: 0;
+            button {
+                width: 10px;
+                height: 10px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use indent variables

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+                border-radius: 0;
+                border: 2px solid @color-maksym__main-black;
+            }
+        }
+    }
+}
+
+/* SECTION__CATEGORIES*/
+.section__categories {
+    margin: -15px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use indent variables

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+    .categories {
+        &__title {
+            .lib-visually-hidden();
+        }
+    }
+
+    [data-content-type="column-line"] {
+        .categories__item {
+            margin: 15px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use indent variables

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+/* SECTION__SLIDER */
+
+.section__slider {
+    .slider {
+        &__title {
+            .lib-visually-hidden();
+        }
+        .pagebuilder-slide-wrapper {
+            display: flex;
+            align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use flex mixins, check it here:
`/lib/web/css/source/lib/_utilities.less : line 412`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+            padding: 0 5px 0 5px;
+            line-height: 30px;
+            font-size: 26px;
+            font-weight: 500px;
+            text-transform: capitalize;
+        }
+        &__item {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use flex mixins

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 15]
```diff
@@ -0,0 +1,233 @@
...
+        &__title {
+            position: absolute;
+            top: 5px;
+            z-index: 1;
+            background-color: white;
+            margin: 0;
+            padding: 0 5px 0 5px;
+            line-height: 30px;
+            font-size: 26px;
+            font-weight: 500px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use font-weight variable

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 16]
```diff
@@ -0,0 +1,233 @@
...
+.section.section__slider {
+    padding-bottom: 30px;
+}
+
+.section {
+    padding-bottom: 30px;
+    h2,
+    h3 {
+        font-size: 24px;
+        font-weight: 600px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Use font-weight variable, check for:
```
//  Weights
@font-weight__light: 300;
@font-weight__regular: 400;
@font-weight__medium: 500;
@font-weight__semibold: 600;
@font-weight__bold: 700;
```

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 17]
```diff
@@ -0,0 +1,233 @@
...
+
+    [data-appearance="collage-right"]
+        .pagebuilder-banner-wrapper
+        .pagebuilder-overlay {
+        max-width: 50%;
+        height: 100%;
+    }
+    [data-element="wrapper"] {
+        display: flex;
+        align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use flex mixins

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 18]
```diff
@@ -0,0 +1,233 @@
...
+}
+.section.section__slider {
+    padding-bottom: 30px;
+}
+
+.section {
+    padding-bottom: 30px;
+    h2,
+    h3 {
+        font-size: 24px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use `.lib-font-size()` mixin, check it here:
`/lib/web/css/source/lib/_utilities.less : line 53`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 19]
```diff
@@ -0,0 +1,233 @@
...
+        font-weight: 600px;
+        letter-spacing: 1px;
+        text-transform: uppercase;
+        color: @color-maksym__main-black;
+    }
+
+    p {
+        font-size: 14px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use font-size mixin

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 20]
```diff
@@ -0,0 +1,233 @@
...
+        line-height: 22px;
+        font-weight: 300px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use font-weight variable

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 21]
```diff
@@ -0,0 +1,233 @@
...
+        letter-spacing: 1px;
+        color: @color-maksym__main-gray;
+    }
+    button {
+        font-size: 14px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use font-size mixin

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 22]
```diff
@@ -0,0 +1,233 @@
...
+        line-height: 22px;
+        font-weight: 300px;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: Please use font-weight variable

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 23]
```diff
@@ -0,0 +1,233 @@
+h1 {
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: wrap your styles into when you need to add styling for all screen sizes:
`& when (@media-common = true) {}`

File: app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml [hunk 24]
```diff
@@ -14,6 +14,9 @@
         </referenceBlock>
 
         <referenceContainer name="title" remove="true"></referenceContainer>
+        <!-- Shop By Brands -->
+
+
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml]: Remove this comment please

**Review by y-mantuliak — CHANGES_REQUESTED** [review]
Change request comments are added, also I have an additional question. 
Why do you need this file? 
`app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_resets.less`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 1 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml (hunk 24 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Theme/layout/default.xml]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 3 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 20 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 21 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 22 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 3 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 5 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 23 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 2 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 4 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 25]
```diff
@@ -1,246 +1,252 @@
...
+
+    .section__slider {
+        max-width: 1920px;
+        .slider {
+            &__title {
+                .lib-visually-hidden();
+            }
+            .pagebuilder-slide-wrapper {
+                .lib-vendor-prefix-display(flex);
+                align-items: center;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: there is a lib for align-items:
`.lib-vendor-box-align()`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 13 above)
Comment (y-mantuliak) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: there is a lib for align-items:
`.lib-vendor-box-align()`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 26]
```diff
@@ -1,246 +1,252 @@
...
+                }
+                .pagebuilder-collage-content {
+                    width: 100%;
+                    .lib-vendor-prefix-display(flex);
+                    justify-content: space-between;
+                }
+            }
+            [data-element="content"] {
+                .lib-vendor-prefix-display(flex);
+                flex-direction: column;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: there is a lib for flex-direction:
`.lib-vendor-prefix-flex-direction();`

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 27]
```diff
@@ -1,246 +1,252 @@
...
+                text-transform: capitalize;
+            }
+            &__item {
+                .lib-vendor-prefix-display(flex);
+                flex-direction: column;
+                align-items: center;
+            }
+            &__image {
+                .lib-vendor-prefix-display(flex);
+                flex-direction: column;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: .lib-vendor-prefix-flex-direction(column);

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less [hunk 28]
```diff
@@ -1,246 +1,252 @@
...
+
+                padding: @indent__xs;
+                line-height: 30px;
+                .lib-font-size(26);
+                font-weight: @font-weight__heavier;
+                text-transform: capitalize;
+            }
+            &__item {
+                .lib-vendor-prefix-display(flex);
+                flex-direction: column;
```
Comment (y-mantuliak) [inline app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: .lib-vendor-prefix-flex-direction(column);

**Review by y-mantuliak — CHANGES_REQUESTED** [review]

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 28 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 27 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 26 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

File: app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less (hunk 25 above)
Comment (Maksym-Lizohub) [reply app/design/frontend/Elogic/maksym/Magento_Cms/web/css/source/_homepage.less]: done

**Review by y-mantuliak — APPROVED** [review]

**Pull request merged**